### PR TITLE
Scrolling to contact form and adding hover styles

### PR DESCRIPTION
**Pull request by jcueli-rebase**
- Link: https://rebaseit-tech.atlassian.net/jira/software/projects/RW/boards/10?selectedIssue=RW-56
- Test proof 🧪 🧪 
 
https://github.com/user-attachments/assets/afbe055c-5c20-4756-9449-e9a0db488ab2

### CODE REVIEW

File: components/atoms/typography/ReTitleSpan.vue [hunk 1]
```diff
@@ -41,4 +48,7 @@ const cssFontSize = computed(() => {
   font-weight: v-bind(fontWeight);
   text-transform: v-bind(textTransform);
 }
+.is-clickable:hover {
```
Comment (jcovello-rebase) [inline components/atoms/typography/ReTitleSpan.vue:51]: isClickeable es nombre de variable, cickleable seria el nombre para una clase

Comment (jcueli-rebase) [reply components/atoms/typography/ReTitleSpan.vue]: Done!

**Review by jcovello-rebase — APPROVED** [review]
CR OK

**Comment by rodrigo-varela** [comment]
Testing: OK

**Pull request merged**